.github/workflows/deploy-snapshots.yml

```diff
@@ -36,4 +36,4 @@ jobs:
         run: mvn -B deploy --file pom.xml
         env:
           BINTRAY_USERNAME: ugandaemr-ci
```

Comment:
@solemabrothers Change this username to a Secret too

Reply:
Done

Reply:
Done